src/screens/DataTable.js

```diff
@@ -192,7 +192,8 @@ const DataTablePage = () => (
               first column will be used. 'pin' indicates that this column should
               not scroll out of view to the left when the table is scrolled
               horizontally. 'plain' = true indicates that the body cells in the
-              column will not apply pad.
+              column will not apply pad. Setting `verticalAlign` within columns
```

Comment:
Just checking whether it is "deprecated" vs. whether it is now available at a table-wide level in addition.

Reply:
Hmm I looked back at Taylor's PR and it looks like she added a comment that it should be deprecated in v3... So not deprecated yet. I will go back and remove the deprecated tuff from the doc.

Reply:
Fixed in latest commit